docs/compression_algorithms/CompressWeights.md

```diff
@@ -61,6 +61,64 @@ nncf_dataset = nncf.Dataset(data_source, transform_fn)
 compressed_model = compress_weights(model, mode=CompressWeightsMode.INT4_SYM, ratio=0.8, dataset=nncf_dataset) # model is openvino.Model object
 ```
 
+- Accuracy of the 4-bit compressed models also can be improved by using AWQ algorithm over data-based mixed-precision algorithm. It is capable to equalize some subset of weights to minimize difference between
+original precision and 4-bit.
+Below is the example how to compress 80% of layers to 4-bit integer with a default data-based mixed precision algorithm and AWQ.
+It requires to set `awq` to `True` additionally to data-based mixed-precision algorithm.
+
+```python
+from datasets import load_dataset
```

Comment:
IMHO: It looks like an example. What do you think about adding the example in https://github.com/openvinotoolkit/nncf/tree/develop/examples/post_training_quantization and add the link to it in the documentation, but documentation will include only example how to turn on awq in `nncf.compress_weights`.

cc` @AlexKoff88 @MaximProshin